RoddyCore/src/de/dkfz/roddy/core/ProjectLoader.groovy

```diff
@@ -494,13 +494,39 @@ class ProjectLoader {
         List<String> errors = []
 
         // Earliest check for valid input and output directories. If they are not accessible or writeable.
-        // Start with the input directory
-        errors += checkDirForReadabilityAndExecutability(analysis.getInputBaseDirectory(), "input")
-        errors += checkDirForReadabilityAndExecutability(analysis.getOutputBaseDirectory(), "output")
 
-        // Out dir needs to be writable
-        if (!FileSystemAccessProvider.instance.isWritable(analysis.getOutputBaseDirectory()))
-            errors << (String) "The output was not writeable at path ${analysis.getOutputBaseDirectory()}."
+        // Check if the configuration values are both set, print out warning if one was set empty.
+        // If one is empty, fill with the other value, if both are empty, abort!
+        // Checks are done before the readability tests because we need to check for the raw configuration values. The next
+        // check will already use translated value
+
+        String valueInDir = analysis.configuration.configurationValues.get(ConfigurationConstants.CFG_INPUT_BASE_DIRECTORY, "").value
```

Comment:
Why "value..."? I see that the evaluation is necessary, for the subsequent fFS checks, but see my comment further down.

Actually, we seem to have some work confusion here. We have configuration values that additionally can be evaluated. The method `getValue()` is responsible for evaluating an value. *Urgh*